src/components/projects-summary/projects-summary.html

```diff
@@ -60,9 +60,9 @@ <h3 class="project-tile-header">
             class="pficon pficon-warning-triangle-o"></span>
         </h3>
         <p class="project-date">
-          <span ng-if="project | displayName : true"><span ng-bind-html="project.metadata.name"></span> &ndash;</span>
+          <span ng-if="project | displayName : true"><span ng-class="{'word-break' : project.metadata.name}" ng-bind-html="project.metadata.name"></span> &ndash;</span>
```

Comment:
Is there a reason to not simply add the word-break class rather than adding it conditionally?